lib/src/main/aidl/com/samsung/android/sdk/penremote/ISpenEventListener.aidl

```diff
@@ -31,5 +31,5 @@ parcelable SpenEvent;
  * The type of event is specified in the call to ISpenRemoteService
  */
 interface ISpenEventListener {
-    void onEvent(out SpenEvent event);
+    void onEvent(in SpenEvent event);
```

Comment:
I'm pretty certain this was correct initially, could you confirm?

Reply:
Keeping it as `out`  made the auto generated interface not even try to read from the parcel and instead just demand an empty constructor. `inout` works too, but required one more function definition and an empty constructor.